Pipfile

```diff
@@ -7,6 +7,7 @@ name = "pypi"
 "rfc5424-logging-handler" = "*"
 daiquiri = "*"
 requests = "*"
+raven = "*"
```

Comment:
why raven? it reads like "Raven is the legacy Python client for Sentry (getsentry.com) — replaced by sentry-python https://sentry.io"

Reply:
Ah, they are fast. That is the way I did it in the previous project :)